### ISSUE TITLE
[BUG] Unable to repay or liquidate

### ISSUE THREAD

**Issue by LeifOnEarth** [alchemix-finance/alchemix-v2-frontend#498]
### Contact Details

leifinthetrees@protonmail.com

### What happened?

For some reason I keep getting the stop sign icon when I try to liquidate my loan or repay it. Screenshot attached showing my wallet is full of WETH ready to pay back. Everything seems to be in order in both cases, but stop icon is all I get. Using both Brave and Chrome. Any thoughts? What am I missing here?

<img width="1074" alt="Screenshot 2023-04-19 at 10 51 52 PM" src="https://user-images.githubusercontent.com/111478875/233270515-3da050ae-f16f-43d5-8b92-ba57b4799171.png">

### What browser were you using?

Chrome

**Comment by LeifOnEarth** [comment]
<img width="1075" alt="Screenshot 2023-04-19 at 10 55 09 PM" src="https://user-images.githubusercontent.com/111478875/233270949-851a3deb-5eef-41e7-bfd2-604368e200fc.png">

**Comment by n4n0GH** [comment]
@LeifOnEarth can you check if the issue is still there on this proposed fix deployment?
https://alchemix-v2-frontend-git-fix-button-states-alchemix.vercel.app/

tl;dr I refactored the way the "disabled" state for that button is being calculated and it should pop up fine now

**Comment by LeifOnEarth** [comment]
Yes thanks that did unlock the 3 main buttons at the top there!

Much appreciated :)

**Comment by LeifOnEarth** [comment]
Any idea where the best place is to swap yvWETH to ETH? It was giving me this error for everything so I took yvWETH over WETH or ETH during withdrawal.

[Screenshot Alchemix 2023-04-21 08-00-39.jpg]
